### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by nburd90**
Added a background-image and got rid of unnecessary code.

### CODE REVIEW

File: home-page.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Beyond the Reel</title>
+
+    <!--Will add an icon-->
+    <link rel="shortcut icon" href="" type="" />
```
Comment (billythakidd04) [inline home-page.html]: If this isn't done, comment it out or delete it and make a todo ticket for later

File: home-page.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  </head>
+
+  <header>
+    <nav>
+      <section>
+        <a href="/about-us.html">About Us</a><br />
+        <a href="/contact.php">Contact</a><br />
+        <a href="/news.html">News</a><br />
+        <a href="/archive.html">Video Archive</a><br />
```
Comment (billythakidd04) [inline home-page.html]: there shouldn't be any need for the slashes in these paths

File: home-page.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      </section>
+    </nav>
+  </header>
+
+  <body>
+    <div class=""></div>
+
+    <div></div>
```
Comment (billythakidd04) [inline home-page.html]: If these are placeholders you should add a comment regarding their use

File: index.html [hunk 4]
```diff
@@ -1,52 +0,0 @@
-<!DOCTYPE html>
```
Comment (billythakidd04) [inline index.html]: Why was this file deleted?

File: styles.css [hunk 5]
```diff
@@ -1,6 +1,11 @@
-header a {
-  text-decoration: none;
+body {
+  background-image: url(../crispy-couscous/crispy-images/pexel-photo.jpeg);
```
Comment (billythakidd04) [inline styles.css]: this url should not need the repo name in it... it should just be relative to the css directory.

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
some updates and some questions

File: index.html (hunk 4 above)
Comment (nburd90) [reply index.html]: I renamed it to home-page so I think git just recognizes as deleted.

File: home-page.html (hunk 3 above)
Comment (nburd90) [reply home-page.html]: This has been identified as a container div.

File: styles.css [hunk 6]
```diff
@@ -1,6 +1,11 @@
-header a {
-  text-decoration: none;
+body {
+  background-image: url(../crispy-images/pexel-photo.jpeg);
```
Comment (billythakidd04) [inline styles.css]: if you take out the `../` does this stop working? I didn't look at the file structure

**Review by billythakidd04 — COMMENTED** [review]
see question about css url

File: styles.css (hunk 6 above)
Comment (nburd90) [reply styles.css]: I won't know until I merge the page and test.

**Review by billythakidd04 — APPROVED** [review]
I still think the images should be in `` tags but that's up to you guys.

**Comment by nburd90** [comment]
@billythakidd04 is it okay if we merge this into the main for the now and then I can make a separate ticket for the background img url to img src change later? 

As long as it works okay for now, I can work on building other parts of the site and take care of it when I have some time. Have to change the CSS around if I go the img tag route.

**Comment by billythakidd04** [comment]
Yes. It's approved, that was just my opinion.

**Comment by nburd90** [comment]
Gotcha. Wanted to double check before closing the pull rqst.

**Comment by billythakidd04** [comment]
I am concerned that @tfrazzini13 didn't review this given how much time he's had though. Either way you can merge.
